lib/datasource/crate/index.ts

```diff
@@ -129,7 +130,7 @@ export class CrateDatasource extends Datasource {
         return `https://crates.io/crates/${lookupName}`;
       case RegistryFlavor.Cloudsmith: {
         // input: https://dl.cloudsmith.io/basic/$org/$repo/cargo/index.git
-        const tokens = info.url.pathname.split('/');
+        const tokens = info.url?.pathname.split('/');
```

Comment:
Is this necessary? It seems to be handled when parsing the URL further up the stack, and throws if `null`

Reply:
👍 Thanks, it's really not necessary